v2/protegrity-data-tokenization/src/main/resources/protegrity_data_tokenization_metadata.json

```diff
@@ -7,42 +7,60 @@
       "label": "GCS location of your data schema",
       "helpText": "Path to data schema (JSON format) in GCS compatible with BigQuery",
       "paramType": "GCS_READ_FILE",
-      "isOptional": false
+      "isOptional": false,
+      "regexes": [
+        "^gs:\\/\\/[^\\n\\r]+$"
```

Comment:
I think we should also remove whitespaces to avoid cases like `gs://bucket       /folder/*.csv`

Reply:
Removed